### ISSUE TITLE
False positives for all comments in a fresh project

### ISSUE THREAD

**Issue by dvdblk** [realm/SwiftLint#3178]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version 
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Comments are not treated as comments and all rules apply to them. This causes the build to fail.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
Linting Swift files at paths 
Linting 'AppCoordinator.swift' (1/6)
Linting 'AppDelegate.swift' (2/6)
Linting 'RootViewController.swift' (3/6)
Linting 'Coordinator.swift' (4/6)
Linting 'BetterpickUITests.swift' (5/6)
Linting 'BetterpickTests.swift' (6/6)
.../Betterpick/AppDelegate.swift:20:1: warning: Line Length Violation: Line should be 120 characters or less: currently 145 characters (line_length)
.../Betterpick/AppDelegate.swift:33:1: error: Line Length Violation: Line should be 120 characters or less: currently 285 characters (line_length)
.../Betterpick/AppDelegate.swift:34:1: warning: Line Length Violation: Line should be 120 characters or less: currently 159 characters (line_length)
.../Betterpick/AppDelegate.swift:38:1: error: Line Length Violation: Line should be 120 characters or less: currently 218 characters (line_length)
.../Betterpick/AppDelegate.swift:44:1: warning: Line Length Violation: Line should be 120 characters or less: currently 155 characters (line_length)
.../Betterpick/AppDelegate.swift:48:1: warning: Line Length Violation: Line should be 120 characters or less: currently 194 characters (line_length)
.../Betterpick/AppDelegate.swift:64:1: warning: Line Length Violation: Line should be 120 characters or less: currently 199 characters (line_length)
.../Betterpick/AppDelegate.swift:69:1: warning: Line Length Violation: Line should be 120 characters or less: currently 123 characters (line_length)
.../Betterpick/AppDelegate.swift:89:1: warning: Line Length Violation: Line should be 120 characters or less: currently 199 characters (line_length)
.../Betterpick/AppDelegate.swift:96:1: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
.../Betterpick/AppDelegate.swift:65:1: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
.../Betterpick/AppDelegate.swift:61:62: warning: Unused Closure Parameter Violation: Unused parameter "storeDescription" in a closure should be replaced with _. (unused_closure_parameter)
.../BetterpickUITests/BetterpickUITests.swift:19:1: warning: Line Length Violation: Line should be 120 characters or less: currently 182 characters (line_length)
.../Betterpick/RootViewController.swift:20:1: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
.../Betterpick/RootViewController.swift:18:1: warning: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)
Done linting! Found 15 violations, 2 serious in 6 files.
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
    ```bash
    $ swiftlint version
    0.39.2
    ```
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
  Project uses cocoapods (`pod 'SwiftLint', '0.39.2'`) and I have a fresh install of swiftlint via `brew install swiftlint`
* Paste your configuration file: I'm using a default configuration. 

```yml
# default swiftlint configuration is used
```

* ~~Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?~~
  If so, paste their relative paths and respective contents.
* Which Xcode version are you using (check `xcodebuild -version`)?
    ```bash
    $ xcodebuild -version
    Xcode 11.4
    Build version 11E146
    ```
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
    ```bash
    $ echo "// Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later." | swiftlint lint --no-cache --use-stdin --enable-all-rules
    <nopath>:1:1: error: Line Length Violation: Line should be 120 characters or less: currently 210 characters (line_length)
    Done linting! Found 1 violation, 1 serious in 1 file.
    ```

EDIT: Added a screenshot

![Xcode screenshot from a new project](https://user-images.githubusercontent.com/10353572/79122869-21472380-7d99-11ea-8937-eb1220eda6dc.png)

To clarify, this also happens in other Swift source files, not just AppDelegate.

**Comment by dvdblk** [comment]
In case this helps someone. All you need to do is add these lines into your **.swiftlint.yml** file:

```
line_length:
  warning: 120
  ignores_function_declarations: true
  ignores_comments: true
  ignores_interpolated_strings: true
  ignores_urls: true
```

This issue could be closed, but I'm curious to know why is this not the default behaviour? Thanks!

**Comment by marcelofabri** [comment]
We don't have control over Apple's templates and SwiftLint instead tries to be opinionated, yet flexible (as you found out).
